snap_plugin/v1/namespace.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+# http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .namespace_element import NamespaceElement
+
+
+class Namespace(object):
+    """Namespace of a metric.
+
+    Args:
+        elements (:obj:`list` of :py:class:`~snap_plugin.v1.namespace_element.NamespaceElement`):
+            namespace elements
+
+    """
+
+    def __init__(self, pb, *elements):
```

Comment:
Most metrics have static NamespaceElements only, I think it would be beneficial to add possibility to instantiate Namespace from a list of strings.